app/src/main/java/com/globant/example/mentorapp/mvp/base/BaseView.java

```diff
@@ -0,0 +1,17 @@
+package com.globant.example.mentorapp.mvp.base;
+
+import com.globant.example.mentorapp.home.presentation.model.ListUsersViewModel;
+
+/**
+ * Interface to define List of users  basic behaviors
+ * Created by oscar.amezquita on 9/06/2017.
+ */
+
+public interface BaseView {
+
+    /**
+     * Method to organize UI actions in view layer
+     */
+    void render(ListUsersViewModel model);
```

Comment:
Will you load a list of users in all your views? Perhaps BaseView is not the proper name for this class then.